### ISSUE TITLE
What is the license for the MIDI files?

### ISSUE THREAD

**Issue by JakesMD** [feross/bitmidi.com#141]
Hi!

In the readme you say the files are free. But can I redistribute them / upload one to my own GitHub repository? Do I need to credit the author? Can I edit a file and redistribute it?

Simply put: Is there a license or copyright on these MIDI files?

**Comment by znmeb** [comment]
Yes - you cannot use these MIDI files without a license. It's fine to download and play them, but the person who sequenced them owns the copyright for the work of sequencing them, and the pieces themselves may still be copyrighted!
